flake.nix

```diff
@@ -28,6 +28,12 @@
       url = "github:anduril/jetpack-nixos";
       inputs.nixpkgs.follows = "nixpkgs";
     };
+    # ghaf reference guest virtual machines
+    # - optionally ghaf can be composed of virtual machines from the same repo with dir
+    # that defines the subdirectory
+    netvm = {
+      url = "github:tiiuae/netvm?rev=b7103ce47b17cd61403b459e72839f38d68204f7";
```

Comment:
There should be `follows` attributes for: nixpkgs, flake-utils, and microvm.

Otherwise, as you can see from the flake.lock file changes, it added second instances of them all, which are be different versions